src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -1,18 +1,21 @@
 import { Component } from 'react'
-import { div, h, span } from 'react-hyperscript-helpers'
+import { div, h, label, span } from 'react-hyperscript-helpers'
 import { ButtonPrimary, Link, spinnerOverlay } from 'src/components/common'
+import { TextInput } from 'src/components/input'
 import Modal from 'src/components/Modal'
 import { Ajax } from 'src/libs/ajax'
 import { bucketBrowserUrl } from 'src/libs/auth'
 import { reportError } from 'src/libs/error'
 import * as Utils from 'src/libs/utils'
+import * as _ from 'lodash/fp'
```

Comment:
We usually import the default, e.g.
```suggestion
import _ from 'lodash/fp'
```
This may also work, but we should keep it consistent.